### DISCUSSION TITLE
OSError: 'NAUTILUS_PATH' environment variable is not set.

### DISCUSSION THREAD

**Discussion by evgrmn** [nautechsystems/nautilus_trader#1721] [Q&A] [answered]
Hello,
I'm trying to run the code from the faststart.ipynb file below:
```Python
import datetime
import os
import shutil
from decimal import Decimal

import fsspec
import pandas as pd

from nautilus_trader.backtest.node import BacktestNode
from nautilus_trader.backtest.node import BacktestVenueConfig
from nautilus_trader.backtest.node import BacktestDataConfig
from nautilus_trader.backtest.node import BacktestRunConfig
from nautilus_trader.backtest.node import BacktestEngineConfig
from nautilus_trader.config import ImportableStrategyConfig
from nautilus_trader.config import LoggingConfig
from nautilus_trader.core.datetime import dt_to_unix_nanos
from nautilus_trader.model.data import QuoteTick
from nautilus_trader.model.objects import Price, Quantity
from nautilus_trader.persistence.catalog import ParquetDataCatalog
from nautilus_trader.test_kit.providers import TestInstrumentProvider

from nautilus_trader.persistence.catalog import ParquetDataCatalog

# You can also use a relative path such as `ParquetDataCatalog("./catalog")`,
# for example if you're running this notebook after the data setup from the docs.
# catalog = ParquetDataCatalog("./catalog")
catalog = ParquetDataCatalog.from_env()
catalog.instruments()
```

but an error occurred:

Traceback (most recent call last):
  File "/home/rmn/1/nuatilus/nautilus_trader/a.py", line 27, in <module>
    catalog = ParquetDataCatalog.from_env()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rmn/1/nuatilus/nautilus_trader/nautilus_trader/persistence/catalog/parquet.py", line 172, in from_env
    raise OSError(f"'{_NAUTILUS_PATH}' environment variable is not set.")
OSError: 'NAUTILUS_PATH' environment variable is not set.

Can anyone help me to avoide this error?

I use Linux Debian 11, Python 3.12.4

**Comment by cjdsellers** [comment, marked as answer]
Hi @evgrmn

You have two options here:
- Set the `NAUTILUS_PATH` environment variable to a path you're happy with to write data for the catalog
- Or, explicitly pass a path like the commented out code, e.g. `ParquetDataCatalog("./catalog")`

Hope that helps!

**Comment by evgrmn** [comment]
Thank you. It works.
